client/src/component/Addvideo.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+
+function AddVideo({ videoData }) {
+  const [showForm, SetShowForm] = useState(false);
+  const [title, setTitle] = useState("");
+  const [url, setUrl] = useState("");
+  const [allData, setAllData] = useState(videoData);
+
+  const handleShowForm = () => {
+    SetShowForm(!showForm);
+  };
+  const handleAddVideo = (event) => {
+    event.preventDefault();
+
+    const newVideoData = {
+      id: Math.floor(Math.random() * 1000),
```

Comment:
This is a nice idea, but we could still run into an issue in which we create multiple video objects with the same id.

Another solution would be to get the last index of the videoData array, and increment the id of this index. so for example `id: videoData[videoData.length -1].id + 1`. Or even better, as this looks quite confusing, we can use lowdash and do `_.last(videoData).id + 1`.

Reply:
Let me know if that doesn't make sense and we can discuss it in our next video call :)